django_slack/backends.py

```diff
@@ -1,15 +1,17 @@
-import pprint
-import urllib
-import urllib2
+from __future__ import print_function
 
+import pprint
+from six.moves import urllib
 from .utils import Backend
 
+
 class Urllib2Backend(Backend):
     def send(self, url, data):
-        r = urllib2.urlopen(urllib2.Request(url, data=urllib.urlencode(data)))
+        r = urllib.urlopen(urllib.Request(url, data=urllib.urlencode(data)))
```

Comment:
Please add a comment that this is using ``six.moves`` - otherwise it looks very very weird that a class called "Urllib2Backend" backend is not using ``urllib2``!